src/main/java/com/questhelper/helpers/quests/perilousmoon/PerilousMoon.java

```diff
@@ -0,0 +1,702 @@
+		buildCamps.addStep(and(buildingSupplies, madePrisonCamp), goBuildEarthCamp);
+		buildCamps.addStep(buildingSupplies, goBuildPrisonCamp);
+		buildCamps.addStep(inAntechamber, takeBuildingSupplies);
+		steps.put(10, buildCamps);
+
+		ConditionalStep goReturnToJessamine = goEnterNeypotzli.copy();
+		goReturnToJessamine.addStep(inAntechamber, talkToJessamineForTalismans);
+		goReturnToJessamine.addStep(inStreambound, leaveStream);
+		goReturnToJessamine.addStep(inEarthbound, leaveEarth);
+		goReturnToJessamine.addStep(inPrison, leavePrison);
+		steps.put(12, goReturnToJessamine);
+		steps.put(13, goReturnToJessamine);
+		steps.put(14, goReturnToJessamine);
+
+		ConditionalStep goTalkToNahta = new ConditionalStep(this, enterCamTorum);
+		goTalkToNahta.addStep(inCamTorum, talkToNahta);
+		goTalkToNahta.addStep(inAntechamber, leaveNeypotzli);
+		steps.put(15, goTalkToNahta);
+
+		ConditionalStep goTalkToBlacksmith = new ConditionalStep(this, enterCamTorum);
+		goTalkToBlacksmith.addStep(inCamTorum, talkToBlacksmith);
+		goTalkToBlacksmith.addStep(inAntechamber, leaveNeypotzli);
+		steps.put(16, goTalkToBlacksmith);
+
+		ConditionalStep goFindEyatlalli = new ConditionalStep(this, enterCamTorum);
+		goFindEyatlalli.addStep(eyatlalliNearby, talkToEyatlalli);
+		goFindEyatlalli.addStep(inNeypotzli, useTalismans);
+		goFindEyatlalli.addStep(inCamTorum, enterNeypotzliAfterBlacksmith);
+		steps.put(17, goFindEyatlalli);
+		steps.put(18, goFindEyatlalli);
+
+		ConditionalStep goTalkToJessamineAfterEyatlalli = new ConditionalStep(this, enterCamTorum);
+		goTalkToJessamineAfterEyatlalli.addStep(inNeypotzli, talkToJessamineAfterEyatlalli);
+		goTalkToJessamineAfterEyatlalli.addStep(inCamTorum, enterNeypotzliAfterBlacksmith);
+		steps.put(19, goTalkToJessamineAfterEyatlalli);
+		steps.put(20, goTalkToJessamineAfterEyatlalli);
+		steps.put(21, goTalkToJessamineAfterEyatlalli);
+		steps.put(22, goTalkToJessamineAfterEyatlalli);
+
+		ConditionalStep getItemsWhenInStream = new ConditionalStep(this, getHerbloreSupplies);
+		getItemsWhenInStream.addStep(and(hadPaste, hadScales, rope), enterSmallEntrance);
+		getItemsWhenInStream.addStep(and(hadPaste, hadScales), getHunterSupplies);
+		getItemsWhenInStream.addStep(and(hadPaste, bream, knife), useKnifeOnBream);
+		getItemsWhenInStream.addStep(and(hadPaste, bigFishingNet), fishBream);
+		getItemsWhenInStream.addStep(and(hadPaste), getFishingSupplies);
+		getItemsWhenInStream.addStep(and(pestleAndMortar, moonlightGrub), useGrubOnPestle);
+		getItemsWhenInStream.addStep(and(pestleAndMortar), collectGrub);
+
+		ConditionalStep goGetItems = new ConditionalStep(this, enterCamTorum);
+		goGetItems.addStep(and(inEarthbound, hadPaste, hadScales, hadTail), leaveEarthboundWithLizard);
+		goGetItems.addStep(and(inNeypotzli, hadPaste, hadScales, hadTail), talkToEyatlalliWithItems);
+		goGetItems.addStep(and(inEarthbound, hadPaste, hadScales, rawMossLizard), useKnifeOnLizard);
```

Comment:
We guide them out of the lizard area, then we could guide them to this entrance as well before prompting them to talk to Eyetlalli.
![image](https://github.com/Zoinkwiz/quest-helper/assets/962989/bd827d6b-7322-4d9b-be89-0a8eab742c8a)